akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1472,7 +1472,7 @@ trait LoggingAdapter {
     formatImpl(t, arg)
   }
 
-  private def formatImpl(t: String, arg: Seq[Any]): String = {
+  def formatImpl(t: String, arg: Seq[Any]): String = {
```

Comment:
Make it 

```
/** INTERNAL API */
@InternalApi private[akka] def ...
```

rather than open up as public

Reply:
Ok, so should we do a copy of akka.event.LoggingAdapter#formatImpl into akka.event.MarkerLoggingAdapter?
Do you see a better solution?

Reply:
No need to copy, marking it `private[akka]` (internally public to Akka classes) should make it available to `MarkerLoggingAdapter`, but still not turn it into a public API that we need to maintain binary compatibility for.

Reply:
Ok, got it, thanks for details.